_field-types/mapping-parameters/format.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: default
+title: Format
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 50
+has_children: false
+has_toc: false
+---
+
+# Format
+
+The `format` mapping parameter specifies the date formats that a date field can accept during indexing. By defining the expected date formats, you ensure that date values are correctly parsed and stored, facilitating accurate search and aggregation operations.
+
+## Example: Defining a custom date format
+
+Create index `events` with `event_date` field configured to a custom date format `yyyy-MM-dd HH:mm:ss` using following request:
+
+```json
+PUT events
+{
+  "mappings": {
+    "properties": {
+      "event_date": {
+        "type": "date",
+        "format": "yyyy-MM-dd HH:mm:ss"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Index a document using matching format `"2025-03-26 15:30:00"` for the `event_date` field:
+
+```
+PUT events/_doc/1
+{
+  "event_name": "Conference",
+  "event_date": "2025-03-26 15:30:00"
+}
+```
+{% include copy-curl.html %}
+
+## Example: Using multiple date formats
+
+Create an index where `log_timestamp` field accepts both the custom date format `yyyy-MM-dd HH:mm:ss` and the `epoch_millis` format using the following request:
+
+```json
+PUT logs
+{
+  "mappings": {
+    "properties": {
+      "log_timestamp": {
+        "type": "date",
+        "format": "yyyy-MM-dd HH:mm:ss||epoch_millis"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Index the first document using date-time string `"2025-03-26 08:45:00"`:
+
+```json
+PUT logs/_doc/1
+{
+  "message": "System rebooted",
+  "log_timestamp": "2025-03-26 08:45:00"
+}
+```
+{% include copy-curl.html %}
+
+Index the second document using the millisecond representation `1711442700000`:
+
+```json
+PUT logs/_doc/2
+{
+  "message": "System updated",
+  "log_timestamp": 1711442700000
+}
+```
+{% include copy-curl.html %}
+
+## Built-in date formats
```

Comment:
Are these the same as the Date built-in formats? If so, remove this table and link to https://opensearch.org/docs/latest/field-types/supported-field-types/date/#built-in-formats